swig/config/ios-arm64-config.jam

```diff
@@ -0,0 +1,10 @@
+import os ;
+
+using darwin : arm64 : clang :
+    <cxxflags>-std=c++17
+    <cxxflags>"-miphoneos-version-min=17.2"
+    <cxxflags>-fno-strict-aliasing
+    <cxxflags>-fvisibility=hidden
+    <cxxflags>-D_DARWIN_C_SOURCE
+    <linkflags>"-isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk"
```

Comment:
I would not hardcode paths here, use environment variables:

e.g.:
```
ANDROID_TOOLCHAIN = [ os.environ ANDROID_TOOLCHAIN ] ;
ANDROID_API = [ os.environ android_api ] ;

using clang-linux : arm : $(ANDROID_TOOLCHAIN)/bin/armv7a-linux-androideabi$(ANDROID_API)-clang++ :
```

I think you already define that path in `build-ios-arm64.sh`

once again DRY principle, otherwise when there's a new path you have to fix 2 files and waste time figuring out why shit doesn't work. Keep it simple.

Reply:
![Screenshot 2023-12-29 at 7 57 55 PM](https://github.com/frostwire/frostwire-jlibtorrent/assets/163977/712bf97e-2d38-49a1-ae7d-83bfd795229e)

Reply:
Thank you for this.
I'm sure we'll make this happen.
You are close.